### ISSUE TITLE
For cs1.6

### ISSUE THREAD

**Issue by mahmodemos** [thisgamesux/linux-csgo-external#17]
hi 

i search for hack cs1.6 for linux "not wine"  and dont find cheat for 1.6 "i find this hack for csgo"
can you make same hack for 1.6 ?

**Comment by bryantwilliam** [comment]
You're kidding right? I don't think someone's going to decide to create a hack for cs 1.6 (which is pretty much a dead game) because you ask in a csgo hack github source repository. I suggest go on a cs hacking forum and ask there.

**Comment by mahmodemos** [comment]
hhhhhh no , not kidding . 
i play cs1.6 until now :p 
can u give me some cs hacking forum  ?

**Comment by bryantwilliam** [comment]
http://www.mpgh.net/forum/105-counterstrike-1-6-hacks-counter-strike-source-hacks/
http://hackforums.net/forumdisplay.php?fid=73

You should close this issue now.

**Comment by Satkiwi** [comment]
I would wan't to use public pre-compiled hacks unless you want to go on VACation in KQLYfornia. Also a large majority of hacks are for windows only.

**Comment by ammardev** [comment]
https://github.com/thisgamesux/linux-csgo-external/issues/16
same issue :cry: 

> You should close this issue now.

No , the first link is not working and I did not find what I want in the second link .
CS1.6 is popular because its working on the cheap and weak devices so we need the hack for it (I found too many ways for windows but nothing for linux)

**Comment by bryantwilliam** [comment]
@3mmarg97 Yeah but asking that question here isn't the place you should be asking it. Also I notice both of you are following each other on github. Coincidence?

**Comment by ghost** [comment]
This hack would work for CS 1.6 if you took the time to change the way it works. The basic way a cs hack works is that it reads the memory and references the items with the offsets to find player positions and draws overlays. If you got this script to read the memory for the process of 1.6 instead of csgo it and worked around the offsets it would probably work.

**Comment by mahmodemos** [comment]
@Oper8or 
honestly , i don't know how to edit it . but i'm trying . 
can you edit it for cs1.6 ?  
if you need link for cs1.6 contact me :)

**Comment by daniel11609** [comment]
Did anyone made it work
